sync_issues_to_jira/push_event.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 Espressif Systems (Shanghai) PTE LTD
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from github import Github
+import os
+import re
+
+def handle_push_event(event):
+    issue_numbers = []
+    for commit in event['commits']:
+        commit_message = commit['message']
+        issue_numbers += parse_commit_message(commit_message)
+    
+    github = Github(os.environ['GITHUB_TOKEN'])
+    repo = github.get_repo(os.environ['GITHUB_REPOSITORY'])
+    for issue in issue_numbers:
+        gh_issue = repo.get_issue(int(issue))
+        if gh_issue.pull_request:
+            update_pull_request(gh_issue.as_pull_request())
+
+def update_pull_request(pull_request):
+    print("Updating %s" % pull_request)
+    if pull_request.state == 'open':
+        print('Pull request is open, nothing to update.')
+        return
+    original_title = pull_request.title
+    # Prepend [Merged] to the pull request title
```

Comment:
Suggest adding a check-and-return if `[Merged]` is already in the title here, to avoid any accidental double-updates

(I think they're unlikely, but weird things do happen sometimes especially if we monitor push events on multiple branches - which we might want to do, because some PRs are merged only on a release branch.)

Reply:
Fixed in https://github.com/espressif/github-actions/pull/14/commits/f84722b187c1712daadc8c63c2959aaace61f984